java/Mekhmet-Efe_Adzhar/src/com/zoolatech/lecture6/tasks/_3/UserMissingException.java

```diff
@@ -0,0 +1,11 @@
+package com.zoolatech.lecture6.tasks._3;
+
+class UserMissingException extends Exception {
+    public UserMissingException(String errorMessage) {
+        super(errorMessage);
+    }
+
+    public UserMissingException() {
+        System.out.println("Error, User not found:");
```

Comment:
it's better to not use sout statements inside of exception, you can just call super to pass the message